core/src/components/title/title.ios.scss

```diff
@@ -55,3 +55,7 @@
 
   text-align: start;
 }
+
+:host(.title-large.ion-cloned-element) {
+  --color: var(--ion-text-color, #fff);
```

Comment:
Should this be:

```
--color: var(--ion-toolbar-color, --ion-text-color, #fff));
```

Reply:
yeah I think so. the regular ion-title inherits its color, so it will first take on whatever the toolbar text color is